### PR TITLE
Es/new test bank

### PR DESCRIPTION
**Pull request by Elena-Shagdurova**

### CODE REVIEW

**Review by Yauheni95 — APPROVED** [review]

File: src/test/java/school/redrover/GroupUnicornsTest.java [hunk 1]
```diff
@@ -267,11 +267,9 @@ public void verificationSocialIconsGitHub2() throws InterruptedException {
             driver.quit();
         }
     }
-    @Ignore
     @Test
-    public void unsuccessfulLoginDigitalBank() {
-        WebDriver driver = new ChromeDriver();
-        try {
+    public void unsuccessfulLoginDigitalBankTest() {
+        WebDriver driver = getDriver();
```
Comment (PeterKuzyk) [inline src/test/java/school/redrover/GroupUnicornsTest.java:272]: We cant use it anymore, please use Sergey D methods instead

File: src/test/java/school/redrover/GroupUnicornsTest.java [hunk 2]
```diff
@@ -284,17 +282,11 @@ public void unsuccessfulLoginDigitalBank() {
...
-        }
+            Assert.assertTrue(errorMsg.isDisplayed(), "Error message is displayed");
     }
-    @Ignore
     @Test
-    public void successfulLoginDigitalBank() {
-        WebDriver driver = new ChromeDriver();
-        try {
+    public void successfulLoginDigitalBankTest() {
+        WebDriver driver = getDriver();
```
Comment (PeterKuzyk) [inline src/test/java/school/redrover/GroupUnicornsTest.java:289]: the same

**Review by PeterKuzyk — CHANGES_REQUESTED** [review]
Please update your test to meet new code standards

**Comment by Elena-Shagdurova** [comment]
hgjhg

**Pull request closed without merging**